noir_stdlib/src/meta/mod.nr

```diff
@@ -2,9 +2,9 @@ use crate::collections::umap::UHashMap;
 use crate::hash::BuildHasherDefault;
 use crate::hash::poseidon2::Poseidon2Hasher;
 
+mod struct_def;
```

Comment:
Unrelated change (sorry): Renamed the type_def module to struct_def now that the `TypeDefinition` type has been renamed to `StructDefinition`.